GeoIP2/DatabaseReader.cs

```diff
@@ -39,9 +39,9 @@ public DatabaseReader(string file, FileAccessMode mode = FileAccessMode.MemoryMa
         /// <param name="file">The MaxMind DB file.</param>
         /// <param name="locales">List of locale codes to use in name property from most preferred to least preferred.</param>
         /// <param name="mode">The mode by which to access the DB file.</param>
-        public DatabaseReader(string file, List<string> locales, FileAccessMode mode = FileAccessMode.MemoryMapped)
+        public DatabaseReader(string file, IEnumerable<string> locales, FileAccessMode mode = FileAccessMode.MemoryMapped)
```

Comment:
I thought maybe some implementations were there for the sake of the JSON serializer. It is the component I blame whenever I don't understand something here, because it is a bit invasive.